public/styles/components/_pagination.scss

```diff
@@ -0,0 +1,62 @@
+.bsds-pagination {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  li {
+    &.bsds-pagination-page,
+    &.bsds-pagination-stepper {
+      margin: 0;
+    }
+
+    &.bsds-pagination-page {
+      [class*='bsds-button'],
+      [class*='bsds-link'] {
+        border: 0;
+        padding: space(sm) space(lg);
+        background: transparent;
+        color: $text-color-primary;
+        text-decoration: none;
+        @include font-family-base-regular;
+        @include font-size($fs-body);
+
+        &:hover {
+          background: transparent;
+          text-decoration: underline;
+        }
+      }
+
+      &.bsds-pagination-page-current {
+        [class*='bsds-button'],
+        [class*='bsds-link'] {
+          @include font-family-base-heavy;
+        }
+      }
+    }
+
+    .bsds-pagination-stepper-action {
+      padding: space(lg);
+    }
+  }
+
+  .bsds-pagination-page-current {
+    [class*='bsds-button'],
+    [class*='bsds-link'] {
+      display: block;
+      transform: translateY(space(xxs)); // Addresses position of bold font compared to regular font
+    }
+  }
+}
+
+@media (min-width: $breakpoint-lg-min) {
+  .bsds-pagination {
+    .bsds-pagination-page-current {
+      [class*='bsds-button'],
+      [class*='bsds-link'] {
+        transform: translateY(0);
```

Comment:
i think this actually makes it look higher to me

Reply:
Looking different between browsers, not sure how to handle this.